deploy/osd-cluster-acks/sts/4.17/config.yaml

```diff
@@ -0,0 +1,12 @@
+deploymentMode: SelectorSyncSet
+selectorSyncSet:
+  matchExpressions:
+  - key: hive.openshift.io/version-major-minor
+    operator: In
+    values: ["4.16"]
+  - key: api.openshift.com/gate-sts <<<<
+    operator: In <<<
+    values: ["4.17"] <<<
```

Comment:
Are these ` <<<` intentional?  CI is somehow happy, but they look like invalid YAML:

```console
$ curl -s https://raw.githubusercontent.com/openshift/managed-cluster-config/467a5f09278f31a2dfbab3ddb7f0fd2b8da5cdb1/deploy/osd-cluster-acks/sts/4.17/config.yaml |  yaml2json 
Traceback (most recent call last):
...
yaml.parser.ParserError: while parsing a block mapping
  in "<stdin>", line 7, column 5
expected <block end>, but found '<scalar>'
  in "<stdin>", line 9, column 22
```

Vs.:

```console
$ curl -s https://raw.githubusercontent.com/openshift/managed-cluster-config/467a5f09278f31a2dfbab3ddb7f0fd2b8da5cdb1/deploy/osd-cluster-acks/sts/4.17/config.yaml | sed 's/ <<<//' | yaml2json 
{
  "deploymentMode": "SelectorSyncSet",
  "selectorSyncSet": {
    "matchExpressions": [
      {
        "key": "hive.openshift.io/version-major-minor",
        "operator": "In",
        "values": [
          "4.16"
        ]
      },
      {
        "key": "api.openshift.com/gate-sts<",
        "operator": "In",
        "values": [
          "4.17"
        ]
      },
      {
        "key": "api.openshift.com/sts",
        "operator": "In",
        "values": [
          "true"
        ]
      }
    ]
  }
}
```

Reply:
Thanks! Good catch!